scripts/configs/obsparam_testfile.yaml

```diff
@@ -1,27 +1,47 @@
+# A healvis observation parameter file
+
+# --- IO Parameters ---
 filing:
   outdir: "./results/test_out"
   outfile_prefix: 'healvis'
   clobber: False
+
+# -- Simulation Parameters ---
+telescope:
```

Comment:
Most of the parameters in the `telescope_config` yaml file have been moved to the `beam` section. In pyuvsim the beam information is specified in a separate file to more cleanly make the beam dictionary, which assigns beam models to individual antennas. Since healvis doesn't support multiple beam models, we can probably save confusion by just moving the `telescope_location` and `telescope_name` entries to the `telescope` section and removing the `telescope_config_name`.